### PR TITLE
Update pull_request_template.md

### PR DESCRIPTION
**Pull request by elyezer**
# Description

Add comments to areas where are description to what to fill, placeholders and checklist.

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor
- [x] Repo maintenance 

## Extra information

This is how you will see when editing it:

Screenshot 2023-01-25 105728

This is how you will see when previewing it without dropping the comments or forgetting to remove them:

Screenshot 2023-01-25 105708

### CODE REVIEW

**Review by ldjebran — APPROVED** [review]
lgtm

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # Description
-
+<!--
 Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+-->
 
-Fixes # (issue)
+Fixes # <!--(issue)-->
```
Comment (de1987) [inline .github/pull_request_template.md:6]:
```suggestion
Fixes # <!--(URL issue)-->
```

Comment (de1987) [reply .github/pull_request_template.md:6]: WDYT?

Comment (elyezer) [reply .github/pull_request_template.md:6]: What about `FIXES: <!-- THEEDGE-NNNN -->`

Comment (elyezer) [reply .github/pull_request_template.md]: Done

**Review by de1987 — APPROVED** [review]

**Review by omaciel — APPROVED** [review]
/lgtm

**Pull request merged**